### DISCUSSION TITLE
Tab Options

### DISCUSSION THREAD

**Discussion by vertigo220** [doublecmd/doublecmd#1973] [Suggestions]
Some additional tab options that would be very useful:

1. Open new tab to a specified location instead of cloning/duplicating the current tab
2. Reopen closed tab (not having this is very problematic, as accidentally closing a tab, which I did middle-clicking to try to duplicate it, can make it a pain to get it back)
3. Configure middle-click options and add more double-click options (e.g. I'd prefer one or the other to clone/duplicate the tab, which isn't currently necessary but would be if # 1 above were implemented, and the other to perhaps move the tab to the other pane or open it in another window)
4. Cycle tabs by MRU instead of left-to-right (this has been mentioned elsewhere, just putting it here for completeness)
5. Allow formatting of tab names. Not only do the lower-case drive letters and lack of "\'s" after them bug my OCD, but different users may want different things here, e.g. full path vs just current folder vs parent and current folder, etc. XYplorer solves this by allowing customization with variables, e..g <drive>:\...<parent><folder>.

**Comment by zxcat** [comment]
just faced 2. so +1 for easy reopen
